charts/spire/charts/spiffe-oidc-discovery-provider/templates/configmap.yaml

```diff
@@ -8,9 +8,8 @@ domains:
   - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}"
   - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}"
   - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}.svc.{{ include "spire-lib.cluster-domain" . }}"
-  {{- if gt (len .Values.config.domains) 0 }}
-  {{- .Values.config.domains | toYaml | nindent 2 }}
-  {{- end }}
+  {{- $jwtIssuer := include "spire-lib.jwt-issuer" . }}
```

Comment:
This will tightly couple the sub chart with the root level chart. If we break the chart out it all of a sudden depends on this global that will not exist when using the chart individually. Basically a dependency cycle.

Reply:
I think this hyperlink moved to wrong line by updates on the code. Was trying to see which part resolves it, but can't relate that in the line linked.

Reply:
Yeah. the review drifted. Try:

https://github.com/spiffe/helm-charts/pull/230/files#diff-789477a4f2a924ca09b7f80a469fc8827159a3d3d8a6281c37018dc16b60c094R18-R21
And 
https://github.com/spiffe/helm-charts/pull/230/files#diff-7be84ce176a12ba3b77f98055916d01a5c97e1688a3a32e361495839b5ef69caR107

Should cause it to still work when split.